### PR TITLE
Highlight property

### LINKED ISSUE #15: 1.13 Ability to highlight a row or column
Example code how to highlight a rows and columns will be provided.

### UI specs: 

image

### PR DESCRIPTION
**Pull request by Toshegg**
Implemented highlight feature, including new demo page.

Fixes #15 

Doesn't work with custom defined columns and templates.

### CODE REVIEW

File: px-data-grid.html [hunk 1]
```diff
@@ -334,15 +343,80 @@
...
+            highlight: {
+              type: Array,
+              value: []
             }
           };
         }
 
         ready() {
           super.ready();
+          this.grid = this.shadowRoot.querySelector('vaadin-grid');
```
Comment (limonte) [inline px-data-grid.html]: should be private. `this._vaadinGrid`?

Comment (Toshegg) [reply px-data-grid.html]: Done.

File: demo/px-data-grid-highlight-demo.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+<!-- Demo DOM module -->
+<dom-module id="px-data-grid-highlight-demo">
+  <template>
+    <custom-style>
+      <style include="px-demo-styles" is="custom-style"></style>
+    </custom-style>
+
+    <!-- Header -->
+    <px-demo-header
+        module-name="px-data-grid-highlight"
```
Comment (limonte) [inline demo/px-data-grid-highlight-demo.html]: module name should be `px-data-grid`

File: demo/px-data-grid-highlight-demo.html [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        description=""
```
Comment (limonte) [inline demo/px-data-grid-highlight-demo.html]: description is missing

File: demo/px-data-grid-highlight-demo.html [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+      <!-- Props -->
+      <px-demo-props slot="px-demo-props" props="{{props}}" config="[[chosenConfig]]"></px-demo-props>
+
+      <!-- Code Editor -->
+      <px-demo-code-editor slot="px-demo-code-editor" props="{{props}}" config="[[chosenConfig]]"></px-demo-code-editor>
+
+      <!-- Component ---------------------------------------------------------->
+      <px-demo-component slot="px-demo-component">
+        <px-data-grid
+            id="grid"
```
Comment (limonte) [inline demo/px-data-grid-highlight-demo.html]: remove, not needed

**Comment by Toshegg** [comment]
Done.

**Comment by Toshegg** [comment]
Done.

**Pull request merged**